src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionjavapatchfilter/CommentsIndentation/context/case3/Test.java

```diff
@@ -0,0 +1,22 @@
+package TreeWalker.CommentsIndentation;
+
+public class Test {
+    public void test() {
+        try {
+        int a;
+            // violation without filter
+        } catch (Exception ex) {}
+    }
+
+    public void test2() {
```

Comment:
Please add a comment above this method and add it to the diff:
```diff
+               // violation without filter
    public void test2() {
```
This should cover METHOD_DEF scope.

Reply:
done